### ISSUE TITLE
AutoETS : ZeroDivisionError 

### ISSUE THREAD

**Issue by iamyihwa** [Nixtla/statsforecast#467]
### What happened + What you expected to happen

1. Bug 

fcst = StatsForecast(df = cur_subset_Y_train_df, 
                    models=[  AutoETS(season_length =12)
                    ], 
                    freq= 'M', 
                    n_jobs=-1)
fcst.fit() 

2.    expected behiavor 

Model to be fit 

3. Logs 
---------------------------------------------------------------------------
ZeroDivisionError                         Traceback (most recent call last)
Cell In[44], line 13
      4 cur_subset_Y_train_df = Y_train_df.loc[each_id]
      6 fcst = StatsForecast(df = cur_subset_Y_train_df, 
      7                     models=[ # AutoARIMA(season_length = season_length),
      8                     AutoETS(season_length = season_length)
   (...)
     11                     #fallback_model = Naive, 
     12                     n_jobs=-1)
---> 13 fcst.fit() 

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/core.py:579, in _StatsForecast.fit(self, df, sort_df)
    577 self._prepare_fit(df, sort_df)
    578 if self.n_jobs == 1:
--> 579     self.fitted_ = self.ga.fit(models=self.models)
    580 else:
    581     self.fitted_ = self._fit_parallel()

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/core.py:73, in GroupedArray.fit(self, models)
     71     for i_model, model in enumerate(models):
     72         new_model = model.new()
---> 73         fm[i, i_model] = new_model.fit(y=y, X=X)
     74 return fm

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/models.py:553, in AutoETS.fit(self, y, X)
    531 def fit(
    532     self,
    533     y: np.ndarray,
    534     X: Optional[np.ndarray] = None,
    535 ):
    536     """Fit the Exponential Smoothing model.
    537 
    538     Fit an Exponential Smoothing model to a time series (numpy array) `y`
   (...)
    551         Exponential Smoothing fitted model.
    552     """
--> 553     self.model_ = ets_f(
    554         y, m=self.season_length, model=self.model, damped=self.damped
    555     )
    556     self.model_["actual_residuals"] = y - self.model_["fitted"]
    557     return self

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/ets.py:1272, in ets_f(y, m, model, damped, alpha, beta, gamma, phi, additive_only, blambda, biasadj, lower, upper, opt_crit, nmse, bounds, ic, restrict, allow_multiplicative_trend, use_initial_values, maxit)
   1270 if stype != "N" and m == 1:
   1271     continue
-> 1272 fit = etsmodel(
   1273     y,
   1274     m,
   1275     etype,
   1276     ttype,
   1277     stype,
   1278     dtype,
   1279     alpha,
   1280     beta,
   1281     gamma,
   1282     phi,
   1283     lower=lower,
   1284     upper=upper,
   1285     opt_crit=opt_crit,
   1286     nmse=nmse,
   1287     bounds=bounds,
   1288     maxit=maxit,
   1289 )
   1290 fit_ic = fit[ic]
   1291 if not np.isnan(fit_ic):

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/ets.py:962, in etsmodel(y, m, errortype, trendtype, seasontype, damped, alpha, beta, gamma, phi, lower, upper, opt_crit, nmse, bounds, maxit, control, seed, trace)
    950 if np_ >= len(y) - 1:
    951     return dict(
    952         aic=np.inf,
    953         bic=np.inf,
   (...)
    959         states=init_state,
    960     )
--> 962 fred = optimize_ets_target_fn(
    963     x0=par,
    964     par=par_,
    965     y=y,
    966     nstate=nstate,
    967     errortype=errortype,
    968     trendtype=trendtype,
    969     seasontype=seasontype,
    970     damped=damped,
    971     par_noopt=par_noopt,
    972     lowerb=lower,
    973     upperb=upper,
    974     opt_crit=opt_crit,
    975     nmse=nmse,
    976     bounds=bounds,
    977     m=m,
    978     pnames=par_.keys(),
    979     pnames2=par_noopt.keys(),
    980 )
    981 fit_par = fred.x
    982 init_state = fit_par[-nstate:]

File /anaconda/envs/jupyter_env/lib/python3.8/site-packages/statsforecast/ets.py:850, in optimize_ets_target_fn(x0, par, y, nstate, errortype, trendtype, seasontype, damped, par_noopt, lowerb, upperb, opt_crit, nmse, bounds, m, pnames, pnames2)
    848 if seasontype == "N":
    849     gamma = 0.0
--> 850 res = nelder_mead(
    851     ets_target_fn,
    852     x0,
    853     args=(
    854         y,
    855         nstate,
    856         switch(errortype),
    857         switch(trendtype),
    858         switch(seasontype),
    859         damped,
    860         lowerb,
    861         upperb,
    862         opt_crit,
    863         nmse,
    864         bounds,
    865         m,
    866         optAlpha,
    867         optBeta,
    868         optGamma,
    869         optPhi,
    870         givenAlpha,
    871         givenBeta,
    872         givenGamma,
    873         givenPhi,
    874         alpha,
    875         beta,
    876         gamma,
    877         phi,
    878     ),
    879     lower=lowerb,
    880     upper=upperb,
    881     tol_std=1e-4,
    882     max_iter=1_000,
    883     adaptive=True,
    884 )
    885 return res

ZeroDivisionError: division by zero

### Versions / Dependencies

1.5.0 

### Reproduction script

from statsforecast.models import AutoETS, AutoARIMA, Naive, SeasonalNaive
temp_df = pd.DataFrame({'unique_id': [1]*32, 'ds': pd.date_range(start = '1919-04-30', periods = 32, freq = 'M'), 
               'values': [  7.,  17.,   7.,  38.,  89., 269.,  33.,  13.,  17.,  10.,  21.,
        61.,  18.,  35.,  19.,  23.,  75., 188.,  62.,  31.,  39.,  10.,
        26.,  16.,  35.,  40.,  26.,  43.,  72., 301.,  69.,  24.]})

fcst = StatsForecast(df = temp_df, 
                    models=[
                    AutoETS(season_length = 12)
                    ], 
                    freq='M', 
                    n_jobs=-1)
fcst.fit() 

### Issue Severity

None

**Comment by iamyihwa** [comment]
Using forecast() and fallback_model , in a way is a way to make the model not fail at least. 

```
from statsforecast.models import AutoETS, AutoARIMA, Naive, SeasonalNaive
temp_df = pd.DataFrame({'unique_id': [1]*32, 'ds': pd.date_range(start = '1919-04-30', periods = 32, freq = 'M'),
'values': [ 7., 17., 7., 38., 89., 269., 33., 13., 17., 10., 21.,
61., 18., 35., 19., 23., 75., 188., 62., 31., 39., 10.,
26., 16., 35., 40., 26., 43., 72., 301., 69., 24.]})


fcst = StatsForecast(df = temp_df,
models=[
AutoETS(season_length = 12)
],
freq='M',
fallback_model = SeasonalNaive(season_length= season_length), # Naive, 
n_jobs=-1)
fcst.forecast(h = 3)
```

**Comment by FedericoGarza** [comment]
hey @iamyihwa! Thank you for sharing the example. We've fixed the issue in #470. We will release a new version soon. In the meantime, installing from the main branch should solve the problem: `pip install "git+https://github.com/nixtla/statsforecast.git@main"`. :)
